src/compile-middleware.ts

```diff
@@ -66,6 +67,11 @@ function isSuccessful(response: Response) {
   return (statusCode >= 200 && statusCode < 300);
 }
 
+export const babelCompileCache = LRU<string>(<LRU.Options<string>>{
```

Comment:
You know, it might not be.  I needed to do it *before* I set the type on `LRU<string>` -- maybe that removes the need...  meh.  I get this if I don't have `<LRU.Options<string>>`
```
Argument of type '{ max: number; length: (n: string, key: string) => number; }' is not assignable to parameter of type 'number'
```

Maybe @types/lru-cache.d.ts declaration is bad.  And it should feel bad.